sled-agent/src/dump_setup.rs

```diff
@@ -279,16 +287,22 @@ impl DumpSetup {
             }
         }
 
+        let (tx, rx) = oneshot::channel();
```

Comment:
Before this PR, sending this message was a notification, but did not wait for anything to complete.

Now, it awaits the `oneshot`, which identifies that the message was processed.